Marlin/ultralcd.cpp

```diff
@@ -169,7 +169,8 @@ uint8_t lcdDrawUpdate = LCDVIEW_CLEAR_CALL_REDRAW; // Set when the LCD needs to
   typedef void (*screenFunc_t)();
 
   // Different types of actions that can be used in menu items.
-  static void menu_action_back();
+  #define menu_action_back(dummy) _menu_action_back()
```

Comment:
There is a dummy parameters that is generated by expansion of macro _MENU_ITEM_PART_1. The original version that uses arg... expands with 0 argument, so there was not problem. New version that uses __VA_ARGS__ expands to at least 1 argument, so I've need macro menu_action_back(dummy) to discard the dummy argument. The resulting code is not changed, only the preprocessor is happy now :-).